lib/ex_unit/lib/ex_unit/case_template.ex

```diff
@@ -35,7 +63,7 @@ defmodule ExUnit.CaseTemplate do
       use ExUnit.Callbacks
 
       import ExUnit.Assertions
-      import unquote(__MODULE__)
+      import unquote(__MODULE__), only: [using: 1, using: 2]
```

Comment:
Only functional change. Avoids importing potentially unnecessary things (like `__proxy__/2`, I can't remember if `__` functions get imported) or stuff we might define in the future.